src/styles/app.css

```diff
@@ -0,0 +1,10 @@
+.app {
+  display: flex;
+  flex-direction: column;
+}
+
+.nasa-logo {
+  margin: auto;
+  height: 180px;
+  width: 450px;
+}
```

Comment:
atm the NASA logo is distorted (squished/stretched) and when you shrink the browsers width it ends up overflowing and messing up your other styling. I would suggest using the `aspect-ratio` property so that you only have to worry about the width (i.e. the height will update automatically with the width). Then you can use a combination of a percentage width and a max-width to have it scale properly as the browser width changes, i.e.
```
.nasa-logo {
  margin: auto;
  aspect-ratio: 5 / 3;
  width: 100%;
  max-width: 450px;
}
```